amplify.yml

```diff
@@ -22,7 +22,7 @@ frontend:
         - yarn -v
         - yarn
         - yarn disable-telemetry
-        - yarn gql
+        - API_URL=https://api.mpdx.org/graphql yarn gql
```

Comment:
I was getting errors because of the phases code in `api.stage.mpdx.org`. (`Value "PRAYER_REQUEST" does not exist in "ActivityTypeEnum" enum.`) So as a temporary workaround, I pointed graphql codegen at production because it doesn't have the phases changes.